sda-auth/info_test.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type InfoTests struct {
+	suite.Suite
+	ClientID  string `json:"client_id"`
+	OidcURI   string `json:"oidc_uri"`
+	PublicKey string `json:"public_key"`
+	InboxURI  string `json:"inbox_uri"`
+}
+
+func TestInfoTestSuite(t *testing.T) {
+	suite.Run(t, new(InfoTests))
+}
+
+func (suite *InfoTests) SetupTest() {
+
+	suite.ClientID = "test_client_id"
+	suite.OidcURI = "test_oidc_uri"
+	suite.PublicKey = "test_public_key"
+	suite.InboxURI = "test_inbox_uri"
+
+}
+
+func (suite *InfoTests) TestGetInfo() {
+
+	info := Info{ClientID: suite.ClientID, OidcURI: suite.OidcURI, PublicKey: suite.PublicKey, InboxURI: suite.InboxURI}
+
+	assert.Equal(suite.T(), suite.ClientID, info.ClientID)
+	assert.Equal(suite.T(), suite.OidcURI, info.OidcURI)
+	assert.Equal(suite.T(), suite.PublicKey, info.PublicKey)
+	assert.Equal(suite.T(), suite.InboxURI, info.InboxURI)
+
+}
```

Comment:
This doesn't test anything, you need to call the `getInfo` function and compare the response

Reply:
You're right, I can't really remember how I thought this would work. I was checking how it could be possible to check an iris router target func and it doesn't look very straightforward. I haven't seen any other place where we actually test the endpoint corresponding func, so I'm inclined to remove this instead.